spec/v2/index.bs

```diff
@@ -1,5 +1,6 @@
 <pre class='metadata'>
 Title: Technical Specifications for PCF Data Exchange (Version 2.1.0)
```

Comment:
Can the title also reference the `VERSION` textmacro?

Reply:
I've tried this, but unfortunately I don't think it's possible, as I get:
```
LINE 1:66 of Title metadata: Found unmatched text macro [VERSION]. Correct the macro, or escape it by replacing the opening [ with &#91;
```
(Even if I put the `VERSION` text macro on the line before.)
As a form of mitigation, I moved the macro to the line after the title.

Reply:
OK. Let's figure out a resolution independent of this PR, please.